server/index.js

```diff
@@ -89,24 +89,21 @@ app.indexRoute = function (root) {
     var renderJs = RENDER_JS && mode !== "nojs";
     var renderSs = RENDER_SS && mode !== "noss";
 
-    // JS bundle rendering.
+    // JS/CSS bundle rendering.
     var bundleJs;
     var bundleCss;
-    if (renderJs) {
-      if (WEBPACK_TEST_BUNDLE) {
-        bundleJs = WEBPACK_TEST_BUNDLE;
-        bundleCss = "http://127.0.0.1:2992/js/style.css";
-      } else if (WEBPACK_HOT) {
-        bundleJs = "http://127.0.0.1:2992/js/bundle.js";
-      } else if (WEBPACK_DEV) {
-        bundleJs = "http://127.0.0.1:2992/js/bundle.js";
-        bundleCss = "http://127.0.0.1:2992/js/style.css";
-      } else {
-        // First file is JS path.
-        var stats = require("../dist/server/stats.json");
-        bundleJs = path.join("/js", stats.assetsByChunkName.main[0]);
-        bundleCss = path.join("/js", stats.assetsByChunkName.main[1]);
-      }
+    if (WEBPACK_TEST_BUNDLE) {
+      bundleJs = renderJs ? WEBPACK_TEST_BUNDLE : null;
+      bundleCss = "http://127.0.0.1:2992/js/style.css";
+    } else if (WEBPACK_HOT || WEBPACK_DEV) {
+      bundleJs = renderJs ? "http://127.0.0.1:2992/js/bundle.js" : null;
+      // TODO(ALEX): What's hot vs. dev difference?
```

Comment:
Actually, now that I'm testing this, I'm realizing that hot mode CSS and `nojs` are incompatible (because the JS injects the CSS).

@ryan-roemer: Should I make a note in the readme that if you want to test `nojs` you've got to use `npm run dev`, not hot?